src/ArborX_LinearBVH.hpp

```diff
@@ -86,9 +86,17 @@ class BasicBoundingVolumeHierarchy
   query(ExecutionSpace const &space, Predicates const &predicates,
         CallbackOrView &&callback_or_view, View &&view, Args &&...args) const
   {
-    ArborX::query(*this, space, predicates,
-                  std::forward<CallbackOrView>(callback_or_view),
-                  std::forward<View>(view), std::forward<Args>(args)...);
+    Details::CrsGraphWrapperImpl::
```

Comment:
Same comment
```C++
Kokkos::Profiling::pushRegion("ArborX::BVH::query");

Reply:
Agree. Named it `query_crs` though, to distinguish from a regular one.